numbat/tests/interpreter.rs

```diff
@@ -227,6 +227,20 @@ fn test_function_inverses() {
 
 #[test]
 fn test_math() {
+    expect_output("solve_quadratic_equation(9.0, -126, 441)", "x₁ = 7; x₂ = 7");
+    expect_output("solve_quadratic_equation(9, -126, 441)", "x₁ = 7; x₂ = 7");
+    expect_output("solve_quadratic_equation(1, 1, 1)", "x₁ = NaN; x₂ = NaN");
```

Comment:
This is obviously something we should implement properly. If b² - 4 a c < 0, we should show "no real-valued solution" or similar. And if b² - 4 a c == 0, we should only show the single solution -b/2a.